In [1]:
cd /btl/projects/SSF/SmartSeq2/MdReports/test/20171130_NexteraDualIndex

In [2]:
cut -f 1,4,10,13,17-18,20,49,55,56 /btl/analysis/SSF/SmartSeq2/MiSeq_QC/SSF-12355/SSF-12355_SmartSeqAnalysisV1.1_1/SSF-12355_SmartSeqAnalysisV1.1_1.1511986452036.MdReport.tsv | sort -nk3 | column -t


sampleName  totalReads  pctPfReadsAligned  pctAdapter  meanInsertSize  medianInsertSize  meanGcContent  duplicationRateOfMapped  genesDetected  rRNArate
D09_S45     169246      0.093686           0.038163    435.925897      455.0             40.0           0.024091827              63             0.008082909
D07_S43     415144      0.106334           0.043416    320.099208      365.0             41.0           0.0902954                51             0.045564912
D06_S42     290676      0.113343           0.045927    396.813554      418.0             40.0           0.072725065              163            0.034925483
H02_S86     203976      0.130535           0.045937    396.483812      453.0             41.0           0.035904754              66             0.015060595
E11_S59     234812      0.167589           0.052165    378.026168      389.0             42.0           0.07913194               307            0.032707017
H12_S96     136498      0.167841           0.070133    441.18532   

In [3]:
zcat /btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/D06_S42_L001_R1_001.fastq.gz  | awk '(NR%4==2)' | cut -c1-11 | sort | uniq -c | sort -r > D06_S42_R1_cut1-11.out

In [5]:
head D06_S42_R1_cut1-11.out

      9 TTGTAGGCTTA
      9 TTCTGATGGTA
      9 TTATTGTACAG
      9 TTATAGAACAG
      9 TTAAAGCTCCA
      9 TGTTGAGCTTG
      9 TGTAATAAGCT
      9 TCCTACAACAG
      9 TCCCAATGGGA
      9 TCACTGAATTC


compare with output for sample with better % align

In [6]:
head G10_S82_R1_cut1-11.out

   2014 GTATCAACGCA
   1680 TATCAACGCAG
   1087 ACGCAGAGTAC
   1056 GGTATCAACGC
    536 GAATAACGCCG
    509 CTATTGGAGCT
    480 GATTAAGAGGG
    463 GTTCAAAGCAG
    460 CTTTAATATAC
    436 GAATAGGACCG


some percentage of reads seem to start with GGTATCAACGCAG (3 of the top 4 hits are related/offset from each other)


see if bbmerge can predict the adaptor for either sample

In [8]:
eval export DK_ROOT="/broad/software/dotkit"; . /broad/software/dotkit/ksh/.dk_init ; use Java-1.8

Prepending: Java-1.8 (ok)


In [9]:
/btl/analysis/ChIPseq/adaptor/bbmap/bbmerge.sh in1=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/G10_S82_L001_R1_001.fastq.gz in2=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/G10_S82_L001_R2_001.fastq.gz outa=G10_S82_adapters.fa

java -Djava.library.path=/btl/analysis/ChIPseq/adaptor/bbmap/jni/ -ea -Xmx1000m -Xmx1000m -cp /btl/analysis/ChIPseq/adaptor/bbmap/current/ jgi.BBMerge in1=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/G10_S82_L001_R1_001.fastq.gz in2=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/G10_S82_L001_R2_001.fastq.gz outa=G10_S82_adapters.fa
Executing jgi.BBMerge [in1=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/G10_S82_L001_R1_001.fastq.gz, in2=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/G10_S82_L001_R2_001.fastq.gz, outa=G10_S82_adapters.fa]

BBMerge version 37.64
Adapters counted: 0
Total time: 1.902 seconds.

Pairs:               	160350
Joined:              	235       	0.147%
Ambiguous:           	160115   	99.853%
No Solution:         	0       	0.000%
Too Short:           	0       	0.000%

Avg Inse

In [10]:
more G10_S82_adapters.fa

>Read1_adapter
N
>Read2_adapter
N


In [11]:
/btl/analysis/ChIPseq/adaptor/bbmap/bbmerge.sh in1=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/D06_S42_L001_R1_001.fastq.gz in2=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/D06_S42_L001_R2_001.fastq.gz outa=D06_S42_adapters.fa

java -Djava.library.path=/btl/analysis/ChIPseq/adaptor/bbmap/jni/ -ea -Xmx1000m -Xmx1000m -cp /btl/analysis/ChIPseq/adaptor/bbmap/current/ jgi.BBMerge in1=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/D06_S42_L001_R1_001.fastq.gz in2=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/D06_S42_L001_R2_001.fastq.gz outa=D06_S42_adapters.fa
Executing jgi.BBMerge [in1=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/D06_S42_L001_R1_001.fastq.gz, in2=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/D06_S42_L001_R2_001.fastq.gz, outa=D06_S42_adapters.fa]

BBMerge version 37.64
Adapters counted: 0
Total time: 1.298 seconds.

Pairs:               	145338
Joined:              	523       	0.360%
Ambiguous:           	144815   	99.640%
No Solution:         	0       	0.000%
Too Short:           	0       	0.000%

Avg Inse

In [12]:
more D06_S42_adapters.fa

>Read1_adapter
N
>Read2_adapter
N


number of fastq reads in my chosen samples

In [13]:
zcat /btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/G10_S82_L001_R1_001.fastq.gz | awk '{s++}END{print s/4}' 

160350


In [14]:
zcat /btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/D06_S42_L001_R1_001.fastq.gz | awk '{s++}END{print s/4}' 

145338


GGTATCAACGCAG reads in G10_S82_R1 is 4600/160350 - not a lot 2.8% BUT 11 bases of 25 is a lot
picard metrics say 3.2% adapter

In [15]:
head D06_S42_R1_cut1-8.out 

   2569 GTATCAAC
   2366 GTACATGG
   1986 TATCAACG
   1255 GGTATCAA
   1174 ACGCAGAG
    507 GAGTACAT
    395 ATCAACGC
    353 GTGTGTGT
    324 CACACACA
    290 GATTAAAC


5,810 of 145338 reads (3.9%) start first 8 bases of read with some variant on GGTATCAACGCAG 
picard metrics say 4.5% adapter

In [16]:
/btl/analysis/ChIPseq/adaptor/bbmap/bbmerge.sh in1=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/D06_S42_L001_R1_001.fastq.gz outa=D06_S42_R1_adapters.fa 

java -Djava.library.path=/btl/analysis/ChIPseq/adaptor/bbmap/jni/ -ea -Xmx1000m -Xmx1000m -cp /btl/analysis/ChIPseq/adaptor/bbmap/current/ jgi.BBMerge in1=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/D06_S42_L001_R1_001.fastq.gz outa=D06_S42_R1_adapters.fa
Executing jgi.BBMerge [in1=/btl/data/MiSeq0/runs/SmartSeq2/171122_M03102_0462_000000000-BCYCH/Data/Intensities/BaseCalls/D06_S42_L001_R1_001.fastq.gz, outa=D06_S42_R1_adapters.fa]

BBMerge version 37.64
Adapters counted: 0
Total time: 0.946 seconds.

Pairs:               	72669
Joined:              	5       	0.007%
Ambiguous:           	72664   	99.993%
No Solution:         	0       	0.000%
Too Short:           	0       	0.000%

Avg Insert:          	37.4
Standard Deviation:  	1.0
Mode:                	37

Insert range:        	36 - 39
90th percentile:     	38
75th percentile:     	37
50th percentile:     	37
25th percentile:     	36
10th percentile:     	0


In [17]:
more D06_S42_R1_adapters.fa


>Read1_adapter
N
>Read2_adapter
N


On lager, run read_QC on G10